### PR TITLE
 Add a toggle for interval matrix indexing (Issue #411)

### PR DESCRIPTION
**Pull request by inthar-raven**

### CODE REVIEW

**Comment by inthar-raven** [comment]
Added radio buttons for "Interval indexing": 0-indexing (default) and 1-indexing.

**Comment by frostburn** [comment]
1. Missing the actual `ref` that would be manipulated by the `v-model`
2. Commits should be squashed into one
3. The value should be persisted in local storage

**Comment by frostburn** [comment]
The commit message should end with `ref #445`.

**Comment by frostburn** [comment]
Or do we actually have duplicate issues? #411 is just fine then.

File: src/views/AnalysisView.vue [hunk 1]
```diff
@@ -100,9 +101,9 @@ const matrix = computed(() => {
         <tr>
           <th></th>
           <th v-for="i of Math.min(scale.size, MAX_SCALE_SIZE)" :key="i">
-            {{ i }}
+            {{ i - 1 + indexing.value }}
           </th>
-          <th>({{ scale.size + 1 }})</th>
+          <th>({{ scale.size + indexing.value }})</th>
```
Comment (frostburn) [inline src/views/AnalysisView.vue:106]: In `<template>` code this is just called `indexing`. Now it shows NaNs.

**Pull request closed without merging**